### PR TITLE
Add the full list of components installed by this profile

### LINKED ISSUE #16: Update README
The readme only describes the `kubernetes-dashboard` component

### PR DESCRIPTION
**Pull request by marccarre**
Preview available here: https://github.com/weaveworks/eks-quickstart-app-dev/blob/issues/16-list-components-in-readme/README.md

Fixes #16.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 This repo contains an initial set of cluster components to be installed and
 configured by [eksctl](https://eksctl.io) through GitOps.
 
+## Components
+
+- ALB ingress controller -- to easily expose services to the World.
+- [Cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) -- to [automatically add/remove nodes](https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-autoscaler-setup/) to/from your cluster based on its usage.
+- [Helm](https://helm.sh/) -- Kubernetes' package manager.
+- [Flux](https://docs.fluxcd.io/en/stable/) and its [Flux-Helm operator](https://github.com/fluxcd/helm-operator-get-started) -- for [GitOps](https://github.com/weaveworks/awesome-gitops), i.e. git-based continuous delivery.
```
Comment (2opremio) [inline README.md]: Flux and Helm are installed as part of the `eksctl install flux` / `eksctl gitops apply`, they are not strictly part of the profile, so I am not sure we should be mentioning them here.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+- [Prometheus](https://prometheus.io/) (its [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/), its [operator](https://github.com/coreos/prometheus-operator), its [`node-exporter`](https://github.com/prometheus/node_exporter), [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics), and [`metrics-server`](https://github.com/kubernetes-incubator/metrics-server)) -- for powerful metrics & alerts.
+- [Grafana](https://grafana.com) -- for a rich way to visualize all metrics via your favourite dashboards.
+- [Kubernetes dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/) -- Kubernetes' standard dashboard.
+- [Fluentd](https://www.fluentd.org/) & Amazon's [CloudWatch agent](https://aws.amazon.com/cloudwatch/) -- for cluster & containers' [log collection, aggregation & analytics in CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs.html).
+- [podinfo](https://github.com/stefanprodan/podinfo) -- our recommended demo application to discover Kubernetes.
```
Comment (2opremio) [inline README.md]: I am not sure we are recommending it hehe. I think that _a toy demo application_ would be better.

File: README.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
...
 
+## Components
+
+- ALB ingress controller -- to easily expose services to the World.
+- [Cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) -- to [automatically add/remove nodes](https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-autoscaler-setup/) to/from your cluster based on its usage.
+- [Helm](https://helm.sh/) -- Kubernetes' package manager.
+- [Flux](https://docs.fluxcd.io/en/stable/) and its [Flux-Helm operator](https://github.com/fluxcd/helm-operator-get-started) -- for [GitOps](https://github.com/weaveworks/awesome-gitops), i.e. git-based continuous delivery.
+- [Prometheus](https://prometheus.io/) (its [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/), its [operator](https://github.com/coreos/prometheus-operator), its [`node-exporter`](https://github.com/prometheus/node_exporter), [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics), and [`metrics-server`](https://github.com/kubernetes-incubator/metrics-server)) -- for powerful metrics & alerts.
+- [Grafana](https://grafana.com) -- for a rich way to visualize all metrics via your favourite dashboards.
```
Comment (2opremio) [inline README.md]: Typo and slight rephrase.

```suggestion
- [Grafana](https://grafana.com) -- for a rich way to visualize metrics via dashboards you can create, explore, and share.
```

**Comment by 2opremio** [comment]
Thanks @marccarre ! Very minor comments which you can optionally address

File: README.md (hunk 1 above)
Comment (marccarre) [reply README.md]: Fair enough. I'll remove them. 👍

File: README.md (hunk 3 above)
Comment (marccarre) [reply README.md]: Applied, thanks a lot! ✨

File: README.md (hunk 2 above)
Comment (marccarre) [reply README.md]: Fixed.

**Pull request merged**